index.bs

```
@@ -801,3 +813,49 @@ When the <dfn method for="FakeXRDevice">setLightEstimate(|init|)</dfn> method is
+  ArrayBuffer depthData;
+  FakeXRRigidTransformInit normDepthBufferFromNormView;
+  float rawValueToMeters;
+  unsigned long width;
+  unsigned long height;
+};
+
+</script>
+
+{{FakeXRDepthSensingDataInit}} dictionary describes the state of the depth sensing data that should be used when returning latest depth information in [=creating a CPU depth information instance=] and [=creating a GPU depth information instance=] algorithms. All keys present in {{FakeXRDepthSensingDataInit}} correspond to the data required to be returned by [=native depth sensing=] capabilities of the device.
+
+{{FakeXRDepthSensingDataInit/depthData}} corresponds to the desired depth buffer that is to be set on native depth information returned from querying the native device. Not setting {{FakeXRDepthSensingDataInit/depthData}} key in the dictionary signals that the returned native depth information should be <code>null</code>.
+
+{{FakeXRDepthSensingDataInit/normDepthBufferFromNormView}} corresponds to the desired [=depth coordinates transformation matrix=] that is to be set on native depth information returned from querying the native device.
+
+{{FakeXRDepthSensingDataInit/rawValueToMeters}} corresponds to the desired conversion factor that is to be set on native depth information returned from querying the native device.
+
+{{FakeXRDepthSensingDataInit/width}} and {{FakeXRDepthSensingDataInit/height}} correspond to the desired dimensions of the depth buffer that are to be set on native depth information returned from querying the native device.
+
+<div class="algorithm" data-algorithm="set-depth-sensing-information">
+
+When the {{FakeXRDevice/setDepthSensingData()}} method is invoked on {{FakeXRDevice}} |device| with |depthSensingData|, run the following steps:
+
+1. If |depthSensingData|'s {{FakeXRDepthSensingDataInit/depthData}} is <code>null</code>, set |device|'s [=FakeXRDevice/depth sensing data=] to <code>null</code> and abort these steps.
```

I'm not the biggest fan of having two ways to clear the data. I think depthData should be required and this should throw if it's not set when calling setDepthSensingData; unless there's a scenario you could imagine wanting to set the other values but to not have depthData.

---

Marked all keys as `required` and changed the steps to throw in case `depthData` is `null`.